ui/shared/utils/constants.js

```diff
@@ -654,6 +656,26 @@ const VEP_SV_CONSEQUENCES = [
     text: 'Promoter',
     value: 'PROMOTER',
   },
+  {
+    description: 'An SV which the SV breakend is predicted to fall in an exon',
+    text: 'Breakend Exonic',
+    value: 'BREAKEND_EXONIC',
+  },
+  {
+    description: 'An SV which is predicted to result in intragenic exonic duplication without breaking any coding sequences',
+    text: 'Intragenic Exon DUP',
+    value: 'INTRAGENIC_EXON_DUP',
+  },
+  {
+    description: 'An SV is predicted to duplicate the transcription start site',
+    text: 'TSS DUP',
+    value: 'TSS_DUP',
+  },
+  {
+    description: 'The duplication SV has one breakpoint in the coding sequence',
+    text: 'Partial Exon DUP',
+    value: 'PARTIAL_EXON_DUP',
```

Comment:
Per the discussion on the ticket, both this field and `INTRAGENIC_EXON_DUP` are mapped to the old `DUP_LOF` field. I think we should replace `DUP_LOF`  in the UI with these new fields now, and then in the backend if either of these new fields are included n search we should also return results with the old `DUP_LOF` annotation, similar to what we did for `MSV_EXON_OVERLAP`

Also, it looks like we removed the `INTERGENIC` consequences in the pipeline, so we should remove the UI filter for those here

Reply:
Also, for `INTRAGENIC_EXON_DUP` lets add a sentence to the description that says `previously called "Loss of function via Duplication"`